### PR TITLE
[improvement](profile) Add table name and predicates

### PR DESCRIPTION
**Pull request by mrhhsg**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Add information of table name and column predicates in profile:
image

And add info strings in `show query profile "/xxxxx"`
image

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
Maybe we can remove the `VconjunctExprTree` in the profile by the way.

File: be/src/exec/olap_scan_node.cpp [hunk 1]
```diff
@@ -663,6 +690,8 @@ Status OlapScanNode::build_olap_filters() {
         }
     }
 
+    _runtime_profile->add_info_string("OlapFilters", olap_filters_to_string(_olap_filter));
```
Comment (yiguolei) [inline be/src/exec/olap_scan_node.cpp:693]: "PushdownPredicate"

**Review by BiteTheDDDDt — DISMISSED** [review]
LGTM

File: be/src/exec/olap_scan_node.cpp [hunk 2]
```diff
@@ -653,6 +656,30 @@ Status OlapScanNode::normalize_conjuncts() {
     return Status::OK();
 }
 
+static std::string olap_filter_to_string(const doris::TCondition& condition) {
+    std::stringstream ss;
```
Comment (HappenLee) [inline be/src/exec/olap_scan_node.cpp:660]: better use `format` to do thing of format output. `std::stringstream` have performance problem in multi thread

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**